spec/paperclip/attachment_spec.rb

```diff
@@ -20,7 +20,7 @@
 
   it "processes :original style first" do
     file = File.new(fixture_file("50x50.png"), 'rb')
-    rebuild_class styles: { small: '100x>', original: '42x42#' }
+    rebuild_model styles: { small: '100x>', original: '42x42#' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.